script.rb

```diff
@@ -1,11 +1,16 @@
+# !/usr/bin/env ruby
 # Purpose : Write a script where asks for a confirmation when deleting a folder if this folder contains a file .important
 # If the user confirms the deletion, the folder is deleted.
 # If the user cancels the deletion, the folder is not deleted.
 # Author: Guillaume MORET
 # ------------------------------------------------------------------------------
 
-# require class folder.class.rb
-require_relative 'folder.class.rb'
+# require class folder.rb
+require 'byebug'
```

Comment:
Byebug is only used for development, it shouldn't be required in your main program